data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/buffer/KafkaBuffer.java

```diff
@@ -70,7 +70,7 @@ public KafkaBuffer(final PluginSetting pluginSetting, final KafkaBufferConfig ka
                        final AcknowledgementSetManager acknowledgementSetManager,
                        final ByteDecoder byteDecoder, final AwsCredentialsSupplier awsCredentialsSupplier,
                        final CircuitBreaker circuitBreaker) {
-        super(kafkaBufferConfig.getCustomMetricPrefix().orElse(pluginSetting.getName()), pluginSetting.getPipelineName());
+        super(kafkaBufferConfig.getCustomMetricPrefix().orElse(pluginSetting.getName()+"buffer"), pluginSetting.getPipelineName());
```

Comment:
What will this name look like?

```
kafkabuffer
```

Is there a better name we can use here?

Reply:
Yeah. Right now the name looks like "kafka". I am changing it to "kafkabuffer"

Reply:
I'm wondering if we can have buffer in the base name to start with.

So for example:

```
my-pipeline.kafka.buffer.readLatency
```

Perhaps by changing

```
public static final String READ_LATENCY = "readLatency";
```

to

```
public static final String READ_LATENCY = "buffer.readLatency";
```

Thoughts?